<?xml version="1.0"?>
<launch>

    <node name="usb_cam" pkg="usb_cam" type="usb_cam_node">
        <param name="video_device"      type="string"   value="/dev/v4l/by-id/usb-Bison_BisonCam_NB_Pro_200901010001-video-index0" />
        <param name="image_width"       type="int"      value="320" />
        <param name="image_height"      type="int"      value="240" />
        <param name="pixel_format"      type="string"   value="mjpeg" />
        <param name="framerate"         type="int"      value="30" />
    </node>

    <node name="image_view" pkg="rqt_image_view" type="rqt_image_view" respawn="false" output="screen" />

    <node name="object_image_tracking" pkg="mfvs" type="mfvs_obj_image_tracking" output="screen" >
        <param name="input_image_topic"     type="string" value="/usb_cam/image_raw" />
        <param name="output_image_topic"    type="string" value="/tracked_frame" />
        <param name="tracker_type"          type="string" value="CSRT" />
    </node>

</launch>